Directory.Build.targets

```diff
@@ -1,2 +1,14 @@
-﻿<Project>
+﻿<Project InitialTargets="EnsureKoreBuildRestored">
```

Comment:
I suppose there's no way to avoid this boiler plate and have it still fire when build.cmd has never been run. It shouldn't matter too much since this should change very rarely.

Reply:
Yeah, there are just some things that have to exist in each repo. I haven't found a good way to remove this boilerplate code.